### PR TITLE
Added Global.Api service and implemented Management of Countries

### PR DESCRIPTION
**Pull request by SebaKas21**

### CODE REVIEW

File: be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.pl.resx [hunk 1]
```diff
@@ -267,4 +267,19 @@
...
+    <value>Pomyślnie usunięto kraj.</value>
+  </data>
+  <data name="CountrySavedSuccessfully" xml:space="preserve">
+    <value>Pomyślnie zapisano kraj.</value>
+  </data>
+  <data name="EditCountry" xml:space="preserve">
+    <value>Edytuj kraj</value>
+  </data>
+  <data name="NavigateToCountriesText" xml:space="preserve">
+    <value>Powróc do krajów</value>
```
Comment (dawiddworak88) [inline be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.pl.resx]: **nitpick:** typo 'c'

File: be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.en.resx [hunk 2]
```diff
@@ -267,4 +267,19 @@
   <data name="RoleSavedSuccessfully" xml:space="preserve">
     <value>Successfully saved role.</value>
   </data>
+  <data name="CountryDeletedSuccessfully" xml:space="preserve">
+    <value>Successfully deleted country.</value>
```
Comment (dawiddworak88) [inline be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.en.resx]: **suggestion:** Country deleted successfully.

File: be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.en.resx [hunk 3]
```diff
@@ -267,4 +267,19 @@
...
+  </data>
+  <data name="CountrySavedSuccessfully" xml:space="preserve">
+    <value>Successfully saved country.</value>
```
Comment (dawiddworak88) [inline be/src/Infrastructure/Localization/Foundation.Localization/Resources/ClientResources.en.resx]: **suggestion:** Country saved successfully.

File: be/src/Project/Services/Global/Global.Api/DependencyInjection/ConfigurationRoot.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+            hcBuilder
+                .AddCheck("self", () => HealthCheckResult.Healthy());
+
+            if (string.IsNullOrWhiteSpace(configuration["ConnectionString"]) is false)
+            {
+                hcBuilder.AddSqlServer(
+                    configuration["ConnectionString"],
+                    name: "global-api-db",
+                    tags: new string[] { "globalapidb" });
```
Comment (dawiddworak88) [inline be/src/Project/Services/Global/Global.Api/DependencyInjection/ConfigurationRoot.cs:46]: **nitpick:** string[] is redundant

File: be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientCountriesApiController.cs [hunk 5]
```diff
@@ -0,0 +1,65 @@
+﻿using Foundation.ApiExtensions.Controllers;
```
Comment (dawiddworak88) [inline be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientCountriesApiController.cs]: **suggestion:** rename to CountriesApiController

File: be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientCountriesApiController.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+using System.Globalization;
+using System.Net;
+using System.Threading.Tasks;
+using System;
+using Seller.Web.Areas.Clients.DomainModels;
+using Seller.Web.Areas.Clients.Repositories.Countries;
+
+namespace Seller.Web.Areas.Clients.ApiControllers
+{
+    [Area("Clients")]
```
Comment (dawiddworak88) [inline be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientCountriesApiController.cs]: **suggestion:** move to "Global" area everywhere

File: be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientCountriesApiController.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            return this.StatusCode((int)HttpStatusCode.OK, new { Message = this.clientLocalizer.GetString("CountryDeletedSuccessfully").Value });
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get(string searchTerm, int pageIndex, int itemsPerPage)
+        {
+            var token = await HttpContext.GetTokenAsync(ApiExtensionsConstants.TokenName);
+            var language = CultureInfo.CurrentUICulture.Name;
+
+            var countries = await this.clientCountriesRepository.GetAsync(
```
Comment (dawiddworak88) [inline be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientCountriesApiController.cs]: **suggestion:** rename to this.countriesRepository etc.

File: be/src/Project/Web/Seller/Seller.Web/Areas/Clients/Controllers/ClientCountryController.cs [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+using System.Threading.Tasks;
+using System;
+using Microsoft.AspNetCore.Authentication;
+using System.Linq;
+using Foundation.Account.Definitions;
+
+namespace Seller.Web.Areas.Clients.Controllers
+{
+    [Area("Clients")]
+    public class ClientCountryController : BaseController
```
Comment (dawiddworak88) [inline be/src/Project/Web/Seller/Seller.Web/Areas/Clients/Controllers/ClientCountryController.cs]: **suggestion:** move to Global area and rename to CountriesController

File: fe/src/project/Seller.Portal/areas/Clients/pages/ClientCountriesPage/ClientCountriesPage.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+import { ThemeProvider } from "@mui/material/styles";
+import GlobalHelper from "../../../../../../shared/helpers/globals/GlobalHelper";
+import Header from "../../../../../../shared/components/Header/Header";
+import Store from "../../../../../../shared/stores/Store";
+import Footer from "../../../../../../shared/components/Footer/Footer";
+import MenuTiles from "../../../../../../shared/components/MenuTiles/MenuTiles";
+import LocaleHelper from "../../../../../../shared/helpers/globals/LocaleHelper";
+import Catalog from "../../../../../../shared/components/Catalog/Catalog";
+
+const ClientCountriesPage = (props) => {
```
Comment (dawiddworak88) [inline fe/src/project/Seller.Portal/areas/Clients/pages/ClientCountriesPage/ClientCountriesPage.js]: **suggestion:** rename to CountriesPage

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]
Almost there :)

File: be/src/Project/Services/Global/Global.Api/DependencyInjection/ConfigurationRoot.cs (hunk 4 above)
Comment (SebaKas21) [reply be/src/Project/Services/Global/Global.Api/DependencyInjection/ConfigurationRoot.cs:46]: string[] is everywhere in configurationRoot for health checks

**Comment by dawiddworak88** [comment]
Hi @SebaKas21 , could you please merge develop to the branch and add open telemetry?

**Review by dawiddworak88 — APPROVED** [review]
Looks good to me 🥇

**Pull request merged**